.github/workflows/image.yaml

```diff
@@ -244,7 +244,7 @@ jobs:
       - name: Install Go
         uses: actions/setup-go@v4
         with:
-          go-version: ^1.16
```

Comment:
I wonder if we really need to do this, I mean currently it just means higher than, so this should be picking the latest version anyway...

Reply:
@Itxaka good question, I wonder if it's better to let it take the latest one or pin it to the one in the `go.mod` file using `go-version-file`